bin/apisix

```diff
@@ -1065,6 +1065,9 @@ local openresty_args = [[openresty  -p ]] .. apisix_home .. [[ -c ]]
                        .. apisix_home .. [[/conf/nginx.conf]]
 
 function _M.start(...)
+
+    local cmd_logs = "mkdir -p " ..  apisix_home .. "/logs"
```

Comment:
I don't think is ​​a good idea to create directories.  What if the user start apisix by sudo?  Does the worker still have permission to write logs?

Reply:
@dabue @membphis

Reply:
> @dabue @membphis

The directory logs is required when the APISIX is running. On the first running, it needs to be created if we keep the logs removed from the root, and should have the appropriate permissions.
when the directory already exists, whether the directory is created by the first running or is from the beginning,  the permission to write logs are same.
I'll do a test to verify it.